src/builtin.elpi

```diff
@@ -119,6 +119,14 @@ type iabs int -> int.
 
 type rabs float -> float.
 
+type (max) int -> int -> int.
+
+type (max) float -> float -> float.
+
+type (min) int -> int -> int.
+
+type (min) float -> float -> float.
```

Comment:
I'm not sure whether they should really be infix.

Reply:
They are not, it's a bug in the pretty printer but has no effect in practice.